fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -135,6 +135,10 @@ public class FedoraPropsConfig extends BasePropsConfig {
     @Value("${fcrepo.cache.webac.acl.timeout.minutes:10}")
     private long webacCacheTimeout;
 
+    @Value("${fcrepo.register.api.banner:true}")
```

Comment:
I feel like we probably should just have this use the same property as in https://github.com/fcrepo/fcrepo/pull/2028, I can't think of a person that would want to only disable this on the HTML UI or the HTTP API and not both.